### DISCUSSION TITLE
执行 ‘nvim' 命令后，安装插件失败

### DISCUSSION THREAD

**Discussion by Dovahkinn** [ayamir/nvimdots#977] [Q&A] [answered]
错误信息是说没有访问权限，但是我本机已经配置了过了 ssh, 公钥也添加到了 github, 本机命令行手动执行 ssh git clone 可以正常下载，
第一次搞，试了两天

- ssh config:
![image](https://github.com/ayamir/nvimdots/assets/17044194/6867cb66-0127-4158-a6f9-87f7bbe55c9a)
- 执行 `nvim` :
![image](https://github.com/ayamir/nvimdots/assets/17044194/460d0e70-c984-4771-9b67-0d22a8000ad6)

**Comment by charliie-dev** [comment, marked as answer]
try change host from github to github.com

**Comment by Dovahkinn** [comment]
谢谢 @[CharlesChiuGit](https://github.com/CharlesChiuGit) 的建议，
有两个原因：
```
Host github.com // 1. 不能是 github
Hostname github.com
User [username]
PreferredAuthentications publickey
IdentityFile ~/.ssh/id_ed25519_np // 2. 使用的证书不能有密码
```
